google-cloud-spanner/src/test/java/com/google/cloud/spanner/connection/ConnectionImplTest.java

```diff
@@ -412,6 +412,140 @@ public void testExecuteSetAutocommitOff() {
     }
   }
 
+  @Test
+  public void testSetAutocommitToTrue_inAutoCommitAndNotInTransaction_noop() {
+    try (ConnectionImpl subject =
+        createConnection(
+            ConnectionOptions.newBuilder()
+                .setCredentials(NoCredentials.getInstance())
+                .setUri(URI)
+                .build())) {
+      assertThat(subject.isAutocommit(), is(true));
+
+      subject.setAutocommit(true);
+
+      assertThat(subject.isAutocommit(), is(true));
+    }
+  }
+
+  @Test
+  public void testSetAutocommitToTrue_inAutoCommitAndInTransaction_noop() {
+    try (ConnectionImpl subject =
+        createConnection(
+            ConnectionOptions.newBuilder()
+                .setCredentials(NoCredentials.getInstance())
+                .setUri(URI)
+                .build())) {
+      assertThat(subject.isAutocommit(), is(true));
```

Comment:
No change needed in this PR, but for next PR: We prefer the use of `assertTrue(..)` and other standard JUnit assertions over `Truth` assertions. The examples using `assertThat(..)` are also leftovers from pre-Java 8 times.

Reply:
Changed to assertTrue everywhere